install/consul/templates/istio.yaml.tmpl

```diff
@@ -23,7 +23,6 @@ services:
           - apiserver
     ports:
       - "8080:8080"
-    privileged: true
```

Comment:
Don’t remove this. This is for docker compose.

Reply:
Sure, I can put this back, but are you positive docker compose needs it? Everywhere docker advises to never run containers with priv true.

Reply:
This is already the case, the init core dump container is not enabled by default. There is also #8043 that deals with core dumps.

I will revert the consul, but do you have any reference that shows this is required? It does not make sense.